Python/Product/Debugger/Debugger.csproj

```diff
@@ -46,6 +46,11 @@
     <Reference Include="envdte80, Version=8.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
     <Reference Include="Microsoft.VisualStudio.ComponentModelHost, Version=$(VSTarget).0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL" />
     <Reference Include="Microsoft.VisualStudio.Debugger.Engine, Version=1.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL" />
+    <Reference Include="Microsoft.VisualStudio.Debugger.Interop.VSCodeDebuggerHost, Version=1.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL" />
+    <Reference Include="Microsoft.VisualStudio.Debugger.InteropA, Version=9.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
+      <HintPath>..\..\packages\Microsoft.VisualStudio.Debugger.InteropA.9.0.21022\lib\Microsoft.VisualStudio.Debugger.InteropA.dll</HintPath>
+      <Private>True</Private>
```

Comment:
Neither of these properties should be needed. We default all references to non-private (since they mostly are), interop types should be embedded, and all the nuget packages get added to the assembly search path automatically.

Setting any of them in the project file is likely to break that (and I know this was what VS added - just clarifying why we need to override them).

Reply:
We don't embed any interop assemblies in this project (note the references below). I don't think that's intentional, though.

Reply:
Yeah, you're right. Embedding interop types isn't actually the default.

General point is, use the default settings until they fail.